app/styles/front.scss

```diff
@@ -708,33 +686,60 @@ legend {
   width: 1px;
 }
 
-input:checked + .radioButton, input:checked + .radioButton:active:hover {
+.inputButton {
```

Comment:
Mmh. Seems to me you just outlined the parts of the interface we keep on thinking we should improve.
If you agree with this observation, then I'd much rather use this opportunity to single these bad parts out, so that we can get rid of both the specific CSS opt-outs and the additional classes once we've improved them, and keep better track of the work left to do.